src/ostorlab/ui/static_files/index.html

```diff
@@ -0,0 +1,12 @@
+<html>
```

Comment:
Is this the final version of index.html?

Reply:
Was a dummy file, Removed it now.